crates/samples/only-bundled-libs/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# copy files to build folder to form a code package.
+
+add_custom_target(build_rust_sample_only-bundled-libs ALL
+    COMMAND ${cargo_exe} build -p only_bundled_libs
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    DEPENDS build_fabric_rust_pal
+)
+
+set(_pkg_root ${CMAKE_BINARY_DIR}/only-bundled-libs_root)
```

Comment:
unused variables can be removed.

Reply:
Sorry, rusty (ha!) on CMake - should have realized those weren't necessary ones since they didn't start with CMAKE_